tests/unit/verifiers/test_ceph.py

```diff
@@ -244,20 +286,26 @@ def test_check_replication_number(
 
     # [min_replication_number=1] verified two ceph-osd unit
     ceph_osd_verifier = CephOsd([model.units["ceph-osd/0"], model.units["ceph-osd/1"]])
-    expected_fail_result = Result(Severity.FAIL, "The minimum number of replicas in "
-                                                 "'ceph-osd' is 1 and it's not safe to"
-                                                 " restart/shutdown 2 units. 0 units "
-                                                 "are not active.")
+    expected_fail_result = Result(
+        Severity.FAIL,
+        "The minimum number of replicas in "
+        "'ceph-osd' is 1 and it's not safe to"
+        " restart/shutdown 2 units. 0 units "
+        "are not active.",
+    )
```

Comment:
this string could be compressed to fewer lines now that the lines start further left.